extensions/amp-story/amp-story-page.md

```diff
@@ -0,0 +1,109 @@
+  </noscript>
+  </amp-anim>
+</figure>
+<figure class="centered-fig">
+  <amp-anim alt="Page 2 example" width="300" height="533" layout="fixed" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/pages-page-2.gif">
+  <noscript>
+    <img alt="Page 2 example" width="200" src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/pages-page-2.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+<table>
+  <tr>
+    <td class="col-fourty"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout">Supported Layouts</a></strong></td>
+    <td>container</td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong>Examples</strong></td>
+    <td><a href="https://amp.dev/documentation/examples/introduction/stories_in_amp">Annotated code example for stories</a></td>
+  </tr>
+</table>
+
+[TOC]
+
+## Example
+
+```html
+<amp-story-page id="cover">
+  <amp-story-grid-layer template="fill">
+    <amp-video layout="fill" src="background.mp4" poster="background.png" muted autoplay></amp-video>
+  </amp-story-grid-layer>
+  <amp-story-grid-layer template="vertical">
+    <h1>These are the Top 5 World's Most...</h1>
+    <p>Jon Bersch</p>
+    <p>May 18</p>
+  </amp-story-grid-layer>
+  <amp-story-grid-layer template="thirds">
+    <amp-img grid-area="bottom-third" src="a-logo.svg" width="64" height="64"></amp-img>
+  </amp-story-grid-layer>
+</amp-story-page>
+```
+
+## Attributes
+
+### id [required]
+
+A unique identifier for the page. Can be used for styling the page and its descendants in CSS, and is also used to uniquely identify the page in the URL fragment.
+
+### auto-advance-after [optional]
+
+Specifies when to auto-advance to the next page.  If omitted, the page will not automatically advance. The value for `auto-advance-after` must be either:
+
+  * A positive amount of [time](https://developer.mozilla.org/en-US/docs/Web/CSS/time) to wait before automatically advancing to the next page
+  * An ID of an [HTMLMediaElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement) or video-interface video whose completion will trigger the auto-advance
+
+For example:
+
+```html
+<amp-story-page id="tokyo" auto-advance-after="1s">
+```
+
+### background-audio [optional]
+
+A URI to an audio file that plays while this page is in view.
+
+For example:
+
+```html
+<amp-story-page id="zurich" background-audio="./media/switzerland.mp3">
+```
+
+## Children (of amp-story-page)
+
+The `<amp-story-page>` component contains one or more [layers](amp-story-grid-layer.md).  Layers are stacked bottom-up (the first layer specified in the DOM is at the bottom; the last layer specified in the DOM is at the top).
```

Comment:
The layers link will no longer work - but we should update this to a list of the children elements and link out to their ref docs

Reply:
Do you know what the right format for that link is?  Should it be an absolute URL on amp.dev?

Reply:
Sorry, I misread this link yesterday - we should actually hold off on this until we know what the file structure will be. Please disregard 😅